assets/components/organisms/footer/footer.scss

```diff
@@ -0,0 +1,252 @@
+  font-size: 1.1rem;
+  font-weight: 800;
+}
+
+.footer-faculties {
+  border-bottom: 1px solid $gray-400;
+
+  @include media-breakpoint-up(md) {
+    flex: 0 0 100%;
+    border: none;
+  }
+
+  &.active .footer-links {
+    padding-top: 0.5rem;
+  }
+
+  .footer-links {
+    display: flex;
+    flex-flow: row wrap;
+
+    @include media-breakpoint-up(md) {
+      display: flex;
+      padding-top: 0.5rem;
+    }
+
+    li {
+      flex: 0 0 50%;
+      position: relative;
+      padding-right: 1.5rem;
+      margin-bottom: 1rem;
+
+      @include media-breakpoint-up(md) {
+        flex: 0 0 20%;
+      }
+
+      &:before {
+        content: ' ';
+        display: block;
+        position: absolute;
+        top: 0.4rem;
+        left: -0.95rem;
+        width: 0.5rem;
+        height: 0.23rem;
+        background: $red;
+      }
+
+      a {
+        margin: 0;
+      }
+
+      strong {
+        display: block;
+      }
+    }
+  }
+}
+
+.footer-legal {
+  display: flex;
+  border-top: 1px solid $gray-200;
+  padding-top: 2rem;
+  flex-flow: column wrap;
+  text-align: center;
+  flex-basis: 100%;
+
+  @extend .text-muted;
+
+  a:not(:hover) {
+    border: none;
+
+    @extend .text-muted;
+  }
+
+  a + a {
+    margin-left: 1rem;
+  }
+}
+
+.footer-links {
+  margin: 0;
+
+  a {
+    display: inline-block;
+    margin: 0.5rem 0;
+    font-size: 0.85rem;
+    line-height: 1.2rem;
+    color: $gray-600;
+    border-bottom: 1px solid transparent;
+
+    @extend .text-small;
+  }
+
+  li {
+    &:before {
+      content: none;
+    }
+  }
+}
+
+.footer-bottom {
+  display: flex;
+}
+
+@include media-breakpoint-down(sm) {
+  .footer-sitemap {
+    margin-bottom: 0;
+  }
+
+  .footer-collapse {
+    .footer-title {
+      $icon-size: 1rem;
+      $icon-size-half: $icon-size/2;
+      $icon-thickness: 2px;
+      $icon-thickness-half: $icon-thickness/2;
+
+      border-top: 1px solid $gray-300;
+      position: relative;
+
+      &:after {
+        content: ' ';
+        display: block;
+        position: absolute;
+        top: calc(50% - #{$icon-thickness-half});
+        right: 0;
+        width: $icon-size;
+        height: $icon-thickness;
+        background: $gray-400;
+        transition: background 0.2s;
+      }
+
+      &:before {
+        content: ' ';
+        display: block;
+        position: absolute;
+        top: calc(50% - #{$icon-size-half});
+        right: calc(#{$icon-size-half} - #{$icon-thickness-half});
+        width: $icon-thickness;
+        height: $icon-size;
+        background: $gray-400;
+        transition: transform 0.3s;
+      }
+    }
+  }
+
+  .footer-social {
+    display: flex;
+    flex-flow: row wrap;
+    justify-content: space-between;
+  }
+
+  .btn {
+    margin: 0 0.4em 0.6rem 0;
+  }
+
+  .footer-links {
+    max-height: 0;
+    overflow: hidden;
+    transition: 0.2s;
+
+    li:last-child {
+      margin-bottom: 1rem;
+    }
+  }
+
+  .footer {
+    .active {
+      .footer-title {
```

Comment:
Group those 3 into a single and less deep `.footer .active .footer-title` or better, something like (to keep things grouped) : 

```scss
.footer-title {
  // [...]
  .footer .acctive &:before { /*...*/ }
  .footer .acctive &:after { /*...*/ }
}

.footer-links {
  // [...]
  .footer .acctive & { /*...*/ }
}
```